docs/docs/intro/intro.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 1
+---
+
+# Welcome to Cyd
+
+Cyd is an open source app for Windows, Mac, and Linux that makes it easy for you to claw back your data from Big Tech platforms. Right now Cyd supports X, but more platforms (*cough Facebook cough*) are coming soon.
+
+Cyd helps you create a local, private backup of your data &mdash; like all of your tweets, retweets, likes, bookmarks, and direct messages. Once you've done this, Cyd helps you choose what data you want to delete from your online account. You can delete it all, or you can be selective, deleting most of it but keeping what went viral.
+
+Cyd can also help you migrate your data from closed platforms into open ones &mdash; for example, you can migrate your tweets from X into Bluesky.
+
+![Screenshot of Cyd](./img/cyd.png)
+
+## Why Delete Your Data from Tech Platforms?
+
+The tech platforms that we all rely on are controlled by a tiny group of powerful billionaires like Elon Musk, Mark Zuckerberg, and Jeff Bezos. It's increasingly clear that they're not working in the interest of their users. You don't need to give them permanent access to your data. Especially not now, while they're cashing in on the AI boom by selling your data to AI companies to train their models, all without compensating you.
+
+The content that you've been posting to social media for years isn't just enriching billionaires, it's also a privacy nightmare. Most people have an endless trail of [OSINT](https://en.wikipedia.org/wiki/Open-source_intelligence) crumbs about them across the internet, waiting to be exploited.
+
+Many of us go out of our way to protect our privacy. We install ad blockers to prevent surveillance capitalists from tracking our every move online. We use encrypted messaging apps like Signal and enable disappearing messages so that we don't have permanent histories of all of our conversations on our phones.
+
+Why should our social media posts, or for that matter our product reviews, ratings, comments, and upvotes, remain on the internet forever?
+
+If you've had an online presence for a long time, it might be a good idea to see what you've posted in the past and delete everything that you don't want out there. This is especially true if you're an activist, a journalist, a parent, or work for a company or non-profit that some people don't like. If you run a business, you might consider [providing this service to your employees](../cyd-for-teams/intro) as a benefit.
```

Comment:
link is broken for me on the prod site

Reply:
I couldn't immediately reproduce this, but then I realized that it only happens when you're at `/docs/intro`, but not `/docs/intro/` (with a trailing slash). So, I'm making it so all the links into docs include a trailing slash.